src/elementary_operators.jl

```diff
@@ -191,14 +192,104 @@ function identity_operator(s::IndsNetworkMap; kwargs...)
   return ITensorNetwork(Op("I"), operator_inds)
 end
 
+" Create an operator bitstring corresponding to the number x"
+function point_to_opsum(s::IndsNetworkMap, x::Number, dim::Int)
+  ttn_op = OpSum()
+  ind_to_ind_value_map = calculate_ind_values(s, x, dim)
+  string_sites = [
+    (ind_to_ind_value_map[only(s[v])] == 1) ? ("Dup", v) : ("Ddn", v) for
+    v in dimension_vertices(s, dim)
+  ]
+  add!(ttn_op, 1.0, (string_sites...)...)
+  return ttn_op
+end
+
+" Create an operator which maps a function to 0 at all points in xs"
+function map_to_zero_operator(
+  s::IndsNetworkMap, xs::Vector, dims::Vector=[1 for _ in xs]; truncate_kwargs...
+)
+  udim = unique(dims)
+  @assert length(udim) <= 2 # TODO: generalize 
```

Comment:
if `udim` isn't used elsewhere lets do this in one line
`@assert length(unique(dims)) <= 2`